ksql-examples/src/main/java/io/confluent/ksql/datagen/AvroConsumer.java

```diff
@@ -58,10 +60,12 @@
       Map<String, Object> serdeProps = new HashMap<>();
       serdeProps.put(KsqlGenericRowAvroSerializer.AVRO_SERDE_SCHEMA_CONFIG, schemaStr);
 
-      final Serializer<GenericRow> genericRowSerializer = new KsqlGenericRowAvroSerializer(null);
+      final Serializer<GenericRow> genericRowSerializer = new KsqlGenericRowAvroSerializer(null,
```

Comment:
`null`, `null`, `null`? This will result in an `NPE` at Line 61 of `KsqlGenericRowAvroSerializer`
As a side note: Why bother passing the `schema` here? It is never used in `KsqlGenericRowAvroSerializer`

Reply:
Updated the parameters.
Schema is used after changes I made yesterday.

Reply:
won't this still result in an NPE? The `schema` is passed to `SchemaUtil.getXXX` which in both cases does `schema.fields()` so NPE? It is also configuring the SchemaRegistryClient as null